### ISSUE TITLE
Can't set scalars on inputObjectType as non-nullable.

### ISSUE THREAD

**Issue by andres-valdes** [prisma/nexus#129]
```javascript
definition(t) {
    t.id('session_id', { nullable: false });
    t.url('url', { nullable: false });
  }
```
Generates the following schema:
```graphql
  session_id: ID!
  url: URL
```
and the following types:
```typescript
  session_id: string; // ID!
  url?: any | null; // URL
```

**Comment by steida** [comment]
I have the same problem using https://github.com/okgrow/graphql-scalars.

**Comment by steida** [comment]
The current workaround is to restore default GraphQL behavior in makeSchema via

```ts
nonNullDefaults: { input: false, output: false },
```

And override it in objectType

```ts
nonNullDefaults: { output: true },
```

**Comment by Nayni** [comment]
Ran into the same issue, currently using the `field` method and define the type by string reference as that one still has the correct config options.

**Comment by tgriesser** [comment]
I believe this is fixed, see #143 just published under version `0.12.0-beta.1`
